src/datastore/fetchingFunctions.ts

```diff
@@ -130,7 +130,7 @@ export const fetchRiddlePreviews = async (
 			riddleLinkIds,
 			user.email!
 		);
-		if (answerDataDoc) {
+		if (answerDataDoc.length > 1) {
```

Comment:
Chápu správně, že pokud by byla `length === 1`, tak nemáme, jak posoudit, jestli je ta hádanka untouched nebo unfinished (a tedy je vždy untouched)?

Už jsem unavený. :D

Reply:
Nicméně to teď podle mě úplně nefunguje

Reply:
Jo mělo být určitě `length > 0` nebo `length >= 1`, opravím :D